package.json

```diff
@@ -70,7 +70,7 @@
     "format": "prettier --write './**/*.{js,jsx,ts,tsx}'"
   },
   "lint-staged": {
-    "src/**/*.{js,jsx,ts,tsx,json,css}": [
+    "*.{js,jsx,ts,tsx,json,css,md}": [
```

Comment:
`lint-staged` will now run on all markdown files, all files that aren't `node_modules`. Jacked this from the docs: https://github.com/okonet/lint-staged#automatically-fix-code-style-with-prettier-for-javascript-typescript-markdown-html-or-css